readme.md

```diff
@@ -132,20 +141,6 @@ class User extends Model
 }
 ```
 
-## Laravel Nova Support
-If you want to use share parent Nova resources with child models, you may register the following provider at the end of the boot method of your NovaServiceProvider:
-
-```php
-class NovaServiceProvider extends NovaApplicationServiceProvider
-{
-    public function boot() {
-        parent::boot();
-        // ...
-        $this->app->register(\Tightenco\Parental\Providers\NovaResourceProvider::class);
-    }
-}
-```
-
```

Comment:
So, I would rather the majority of people using this not have to do anything, but force the Nova people to have to add some code - not the other way around.

I realize this complexity is also for adding parent scopes to children, so I'll keep reading.

Reply:
Removing the additional Nova provider was actually a by-product of adding a service provider.